config-linux.md

```diff
@@ -330,14 +330,19 @@ The following parameters can be specified to set up the controller:
 
 * **`weight`** *(uint16, OPTIONAL)* - specifies per-cgroup weight. This is default weight of the group on all devices until and unless overridden by per-device rules.
 * **`leafWeight`** *(uint16, OPTIONAL)* - equivalents of `weight` for the purpose of deciding how much weight tasks in the given cgroup has while competing with the cgroup's child cgroups.
-* **`weightDevice`** *(array of objects, OPTIONAL)* - specifies the list of devices which will be bandwidth rate limited. The following parameters can be specified per-device:
+* **`weightDevice`** *(array of objects, OPTIONAL)* - specifies the list of devices which will be weight division of bandwidth. The following parameters can be specified per-device:
```

Comment:
> specifies the list of devices which will be weight division of bandwidth

This reads a bit odd to me -- perhaps "specifies the list of devices which will be bandwidth-limited by weight" ?

I don't understand why we're fixating on "weight division of bandwidth" here -- the relevant kernel doc (https://www.kernel.org/doc/Documentation/cgroup-v1/blkio-controller.txt) only uses that phrase once.